apps/charterafrica/src/components/OrgAndContributor/OrgAndContributor.js

```diff
@@ -0,0 +1,171 @@
+import { Section, RichTypography } from "@commons-ui/core";
+import { Figure } from "@commons-ui/next";
+import { Grid, SvgIcon, Link, Box } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+import GithubIcon from "@/charterafrica/assets/icons/Type=github, Size=24, Color=CurrentColor.svg";
+import EmailIcon from "@/charterafrica/assets/icons/Type=mail, Size=24, Color=CurrentColor.svg";
+import TwitterIcon from "@/charterafrica/assets/icons/Type=twitter, Size=24, Color=CurrentColor.svg";
+import ToolCard from "@/charterafrica/components/ToolCard";
+
+const SocialIcon = React.forwardRef(function SocialIcon(props, ref) {
+  const { href, variant } = props;
+  const icons = {
+    twitter: TwitterIcon,
+    github: GithubIcon,
+    email: EmailIcon,
+  };
+  return href ? (
+    <Grid ref={ref} item>
+      <Link href={href}>
+        <SvgIcon
+          inheritViewBox
+          component={icons[variant]}
+          sx={{
+            color: "text.primary",
+            display: "inline-flex",
+            fill: "none",
+            height: 32,
+            width: 32,
+          }}
+        />
+      </Link>
+    </Grid>
```

Comment:
```suggestion
    <Link href={href}>
      <SvgIcon
        inheritViewBox
        component={icons[variant]}
        sx={{
          color: "text.primary",
          display: "inline-flex",
          fill: "none",
          height: 32,
          width: 32,
        }}
      />
    </Link>
```

Not a good idea to start component with `<Grid item`. How do you ensure that the user will always include it in `<Grid container`?